src/resources/parser/glb-parser.js

```diff
@@ -327,7 +513,7 @@ Object.assign(pc, function () {
                     // don't support 32bit index data
                     indexFormat = pc.INDEXFORMAT_UINT32;
                 }
-                var numIndices = indices.length;
+                numIndices = indices.length;
```

Comment:
can probably remove numIndices here (or make line 519 also use it).

Reply:
done